libs/wordpress-plugin/plugin/trunk/block-protocol.php

```diff
@@ -171,38 +171,44 @@ function block_dynamic_render_callback($block_attributes)
 	);
 }
 
-// Admin-side setup for the block:
-// 1. register the category for the block
-// 2. register the editor script + dependencies
-// 3. register the plugin block
-// 4. contact the BP API and fetch BP blocks
-//    - enqueue block data for adding to the frontend
-//    - register each BP block as a variation of the plugin block
+// Add the block category for our block
```

Comment:
This and following changes in this file splits out the basic block registration from loading the scripts needed in the editor view – the scripts being loaded in non-editor pages is what was causing the console warning about an invalid block type. Script enqueing is moved to an editor-specific hook